test/db/anal/mips

```diff
@@ -46,7 +46,7 @@ axt str.busybox
 EOF
 EXPECT=<<EOF
             0x00406228      fcc38424       addiu a0, a0, -0x3c04       ; 0x44c3fc ; "busybox" ; str.busybox
-(nofunc); str.busybox 0x406228 [DATA] addiu a0, a0, -0x3c04
+(nofunc); str.busybox 0x406228 [DATA] addiu a0, a0, -str.busybox
```

Comment:
Thats not correct. Even if we eant to have such symbol in the instruction like IDA do. It is imho a confusing translation because str.busybox is not at 0x3c04. But at a0-0x3c04

This is why its shown in a comment in the right. If you do so it would be more correct to add pseudoinstructions like mov a0, str.busybox because the - sign is also looking bad in thst representation to me

Reply:
This is what IDA Pro shows:
```
.text:004061F4  # Attributes: noreturn
.text:004061F4
.text:004061F4  # int __cdecl main(int argc, const char **argv, const char **envp)
.text:004061F4                 .globl main
.text:004061F4 main:                                    # DATA XREF: LOAD:00401E44↑o
.text:004061F4                                          # _ftext+1C↑o ...
.text:004061F4
.text:004061F4 var_s0          =  0
.text:004061F4 var_s4          =  4
.text:004061F4 var_s8          =  8
.text:004061F4
.text:004061F4                 addiu   $sp, -0x28
.text:004061F8                 sw      $s1, 0x18+var_s4($sp)
.text:004061FC                 li      $a0, 0xFFFFFFFF
.text:00406200                 move    $s1, $a1
.text:00406204                 li      $a1, 0x2000
.text:00406208                 sw      $ra, 0x18+var_s8($sp)
.text:0040620C                 jal     _mallopt
.text:00406210                 sw      $s0, 0x18+var_s0($sp)
.text:00406214                 li      $a0, 0xFFFFFFFD
.text:00406218                 jal     _mallopt
.text:0040621C                 li      $a1, 0x7F00
.text:00406220                 lui     $a0, 0x45  # 'E'
.text:00406224                 jal     sub_405D30
.text:00406228                 la      $a0, aBusybox    # "busybox"
.text:0040622C                 lw      $a0, 0($s1)
.text:00406230                 lui     $s0, 0x45  # 'E'
.text:00406234                 sw      $a0, dword_457CBC
.text:00406238                 lbu     $v1, 0($a0)
.text:0040623C                 li      $v0, 0x2D  # '-'
.text:00406240                 bne     $v1, $v0, loc_40624C
.text:00406244                 addiu   $v0, $a0, 1
.text:00406248                 sw      $v0, dword_457CBC
.text:0040624C
.text:0040624C loc_40624C:                              # CODE XREF: main+4C↑j
.text:0040624C                 jal     sub_406338
.text:00406250                 lw      $a0, dword_457CBC
.text:00406254                 jal     _getuid
```

Reply:
My fix was just to make the behavior consistent between `scr.color=0` and `scr.color=1`.
Also, this only happens for the xref commands if subrel is true; normal disassembly shows `-0x3c04`

Reply:
I think @trufae has a point but it's out of the scope of this PR and should be thought over and addressed separately from this PR.

Reply:
guess so, it's useful and we can have the original instruction by disabling the asm.sub stuff, mips support is being improved quite a lot recently. love it :)